### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by realcodywburns**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+Security alerts related to the hosting and delivery of the ecips.ethereumclassic.org web site will be addressed and will be taken by either the ecip editors or org admins. Security concerns with I individual ecips should be addressed in the `discussion to` locations.
```
Comment (sorpaas) [inline SECURITY.md]: > Security concerns with I individual ecips should be addressed in the `discussion to` locations.

This sentence doesn't sound right.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Security alerts will not be made publicly until mitigating solutions are identified. Once a remedy is found, dependably will automate the repair process.
+
+This site is free and open source in accordance with the site license which can be found at LICENSE.md or COPY.md
```
Comment (sorpaas) [inline SECURITY.md]: I don't think the licensing issue belongs here. I would suggest removing this avoiding any possibility to get this controversial.

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master  | :white_check_mark: |
+
+
+## Reporting a Vulnerability
+
+Please contact security@ethereumclassic.org with any website or ecip related security concerns.
```
Comment (sorpaas) [inline SECURITY.md]: This section should include all editors and org admins' PGP keys. Here's mine: https://github.com/sorpaas.gpg

File: SECURITY.md (hunk 2 above)
Comment (realcodywburns) [reply SECURITY.md]: I didn't think about that. This is to cover the website.

File: SECURITY.md (hunk 3 above)
Comment (realcodywburns) [reply SECURITY.md]: I can link to this repo https://github.com/ethereumclassic/volunteer/tree/master/Volunteer-Public-Keys

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+Security concerns with each individual ECIP should be addressed to the ECIP authors, the specified `discussion-to` locations, or ECIP editors. Note that when an ECIP is already deployed, it is recommended to talk privately with the ECIP authors or ECIP editors first.
+
+Security alerts related to the website source code will be addressed by ECIP editors. Security concerns related to hosting and delivery of the ecips.ethereumclassic.org website will be addressed by either org admins or ECIP editors. They will not be made publicly until mitigating solutions are identified. Once a remedy is found, part of the repair process may be automated by [Dependabot](https://dependabot.com/).
```
Comment (mikeyb) [inline SECURITY.md]: `They will not be made publicly`

Should this be `They will not be made public`?

**Pull request merged**